romana-install/roles/cache/apply/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# OS Files
+- include: "{{ item }}"
+  with_first_found:
+    - files:
+      - "repo_{{ ansible_distribution|lower }}_{{ ansible_distribution_release|lower }}.yml"
```

Comment:
It's a templated filename. Combined with the `skip: true`, the include task is skipped if it doesn't exist. Since this is caching, it's OK if it is skipped -- we can add additional distro/release combinations later.
The alternative is doing a `stat` first, registering the variable, then having an `include` task with a `when` clause checking that it existed.